### PR TITLE
DBAAS 157 set operator version dynamically in d baa s dynamic console plugin

### PR DESCRIPTION
**Pull request by yzhao583**
Fetch and parse operator name and version from CSV

JIra: [https://issues.redhat.com/browse/DBAAS-157](https://issues.redhat.com/browse/DBAAS-157)

### CODE REVIEW

File: oc-manifest.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
               mountPath: /var/serving-cert
           env:
             - name: DBAAS_OPERATOR_VERSION
-              value: 'dbaas-operator.v0.0.1'
+              value: 'dbaas-operator.v0.1.1'
```
Comment (redhatHameed) [inline oc-manifest.yaml:44]: do we still need this considering we are fetching from CSV?

**Review by redhatHameed — COMMENTED** [review]
@yzhao583 minor comments else LGTM

File: oc-manifest.yaml (hunk 1 above)
Comment (yzhao583) [reply oc-manifest.yaml:44]: @redhatHameed Since we have already added the env variable, maybe we can use it as a back-up? Or I can remove it

Comment (redhatHameed) [reply oc-manifest.yaml:44]: Okay, I am fine with that if you want to show the operator version in the environment variable.

**Comment by redhatHameed** [comment]
not sure but I am still seeing the same issue 
Screen Shot 2021-09-23 at 4 55 38 PM

**Comment by yzhao583** [comment]
@redhatHameed Did you pull the latest change, and rebuild the image? Looks like the latest commit is not there

**Review by redhatHameed — APPROVED** [review]
@yzhao583 Thanks it worked and lgtm

**Comment by redhatHameed** [comment]
@jeremyary can we merge this if you don't have any further comments thanks

**Pull request merged**